packages/enhanced/src/lib/container/runtime/FederationRuntimePlugin.ts

```diff
@@ -151,16 +153,9 @@ class FederationRuntimePlugin {
     modifyEntry({
       compiler,
       prependEntry: (entry) => {
-        Object.keys(entry).forEach((entryName) => {
-          const entryItem = entry[entryName];
-          if (!entryItem.import) {
-            // TODO: maybe set this variable as constant is better https://github.com/webpack/webpack/blob/main/lib/config/defaults.js#L176
-            entryItem.import = ['./src'];
-          }
-          if (!entryItem.import.includes(entryFilePath)) {
-            entryItem.import.unshift(entryFilePath);
-          }
-        });
+        entry['federation-runtime'] = {
```

Comment:
don't manipulate entypoints of user code, just add a new entry file and `integrateChunk`